src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -151,60 +172,70 @@ const SavedAddresses: React.FC<SavedAddressesProps> = props => {
     const isDefaultAddress = address.isDefault
 
     return (
-      <BorderBox
-        p={2}
-        width="100%"
-        flexDirection="column"
-        key={"addressIndex" + index}
-      >
-        <SavedAddressItem
-          index={index}
-          address={address}
-          handleClickEdit={() => handleEditAddress(address, index)}
+      <>
+        {" "}
+        <ToastComponent
+          showNotification={notificationState.notificationVisible}
+          notificationAction={notificationState.action}
+          duration={5000}
+          title="Address Successfully"
+          onCloseToast={onCloseToast}
         />
-        <Separator my={1} />
-        <ModifyAddressWrapper>
-          {!isDefaultAddress && (
+        <BorderBox
+          p={2}
+          width="100%"
+          flexDirection="column"
+          key={"addressIndex" + index}
+        >
+          <SavedAddressItem
+            index={index}
+            address={address}
+            handleClickEdit={() => handleEditAddress(address, index)}
+          />
+          <Separator my={1} />
+          <ModifyAddressWrapper>
+            {!isDefaultAddress && (
+              <Box mr={[3, 1]}>
+                <Text
+                  onClick={() => handleSetDefaultAddress(address.internalID)}
```

Comment:
All non button / input elements that have mouse events like this should be wrapped in [Clickable](https://palette.artsy.net/components/clickable).

Reply:
Disappointed the a11y linter isn't picking this up

Reply:
I wonder if its because the [`styled-components`](https://github.com/artsy/force/blob/master/.eslintrc.js#L16) scope is too narrow?

Reply:
We are using this though https://github.com/brendanmorrell/eslint-plugin-styled-components-a11y